nsls2forge_utils/dashboard.py

```diff
@@ -5,6 +5,35 @@
 functions had to be re-implemented here.
 """
 from .all_feedstocks import get_all_feedstocks
+from .meta_utils import get_attribute
+
+
+def _extract_org(pkg):
+    try_dev_url = False
+    # get org from home url
+    home_str = get_attribute('about home', pkg, 'nsls-ii-forge')
+    if home_str is None:
+        try_dev_url = True
```

Comment:
Maybe make `try_dev_url = True` unconditional, and check 2 conditions below, such as:
```py
try_dev_url = True
if home_str is not None and 'github.com' in home_str:
    ...
    try_dev_url = False
```

I think it will be a bit cleaner and clearer, and will not require the `else` condition with the same actions as in the first condition.